### DISCUSSION TITLE
Error: Maximum call stack size exceeded

### DISCUSSION THREAD

**Discussion by Ani-MBC** [AntlerVC/firetable#830] [Support / Q&A] [answered]
**Describe the bug**

**To Reproduce**
1. Go to 'templates list'
2. Click on 'multiple templates & view them'
3. Click on "start from scratch" (without template)

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/53514554/191221025-b8ba7a14-9adb-462b-ba64-4a51307e45b6.png)

**Rowy Run version**
Date & Time: 20th Sept 2022, 1.50 PM IST

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
Desktop 
- Chrome
-

**Comment by bofeiw** [comment, marked as answer]
I am able to re-create this bug, we are working on a fix.



https://user-images.githubusercontent.com/34177142/193455707-f840d889-a782-4420-9438-f97f5cb2a5d0.mov
